scripts/gosec.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -o pipefail
+
+if [[ -x "$(command -v gosec)" ]]; then
+  find cmd pkg -type d -print0 | xargs --null gosec -severity=medium
```

Comment:
can we test all severity levels?

Reply:
Sure we can, doing so in the next revision.